packages/bundle-analyzer/src/index.ts

```diff
@@ -35,39 +36,43 @@ import { getAssets } from "./assets";
  *   enableBundleAnalysis: true,
  *   debug: true,
  * };
- * const bundle-analyzerOpts = {
+ * const bundleAnalyzerOpts = {
  *   beforeReportUpload: async (original) => original,
+ *   ignorePatterns: ["*.map"],
+ *   normalizeAssetsPattern: "[name]-[hash].js",
  * };
  *
- * createAndUploadReport(buildDir, coreOpts, bundle-analyzerOpts)
+ * createAndUploadReport(buildDirs, coreOpts, bundleAnalyzerOpts)
  *   .then(() => console.log('Report successfully generated and uploaded.'))
  *   .catch((error) => console.error('Failed to generate or upload report:', error));
  */
 export const createAndUploadReport = async (
-  buildDirectoryPath: string,
+  buildDirectoryPaths: string[],
```

Comment:
What do you think about making this `string | string[]`?

Might be excess complexity though, I'm fine with sticking with the `string[]`.

Reply:
Yeah, I had turned that over in my head, too... I think it would be easy to extend to allow `string | string[]` in the future so thought it'd be good to keep the simple case until can get some more feedback